Source/ConfigurationProfile/Get-IntuneConfigurationProfileSettings.ps1

```diff
@@ -43,19 +43,25 @@ function Get-IntuneConfigurationProfileSettings
         $graphVersion = "beta"
     }
     process {
-        
 
-        $response = Invoke-MgRestMethod -Method Get -Uri "$uri/$graphVersion/deviceManagement/configurationPolicies('$Id')/settings" -OutputType Json | ConvertFrom-Json
+        $configurations = @()
+        $reqeustUri     = "$uri/$graphVersion/deviceManagement/configurationPolicies('$Id')/settings"
 
+        do{
+            $response = Invoke-MgRestMethod -Method Get -Uri $reqeustUri -OutputType JSON | convertFrom-json
```

Comment:
Let's add '-Depth 50' here to ensure that we don't miss anything with future values that may come back.